MinPlatformPkg/Test/Library/TestPointCheckLib/PeiCheckFvInfo.c

```diff
@@ -61,11 +61,9 @@ TestPointCheckFvInfo (
       "  BA=%08x  L=%08x  Format={%g}",
       (UINT32)(UINTN)FvInfo->FvInfo,
       FvInfo->FvInfoSize,
-      &FvInfo->FvFormat,
-      FvInfo->ParentFvName,
-      FvInfo->ParentFileName
+      &FvInfo->FvFormat
       ));
-    if (FvInfo->ParentFvName != NULL) {
+    if (FvInfo->ParentFvName != NULL) 
```

Comment:
I haven't looked at the compiler results, but I'd imagine this removal of this opening brace will be an issue.